src/types.ts

```diff
@@ -353,6 +353,10 @@ export type ExportChannelResponse = {
   task_id: string;
 };
 
+export type ExportUsersResponse = {
```

Comment:
is there a difference or will be in future between this and `ExportChannelResponse`?

Reply:
Currently there is no difference, but wanted to keep it separate just in case we decide to diverge in the future.